scripts/rf_bios_settings.py

```diff
@@ -81,5 +102,8 @@
     print( e )
 finally:
     # Log out
-    redfish_obj.logout()
+    try:
```

Comment:
Is there a need to wrap logout with try/except handling? I don't think we want to necessarily suppress failed logout attempts since the user would be unaware of a stale session left behind.

Reply:
I just want to skip failed logout

Reply:
failed logout will throw "Invalid session resource" error

Reply:
how to handle this case: 

ami redfish server not allow PasswordChangeRequired account session logout it self session

it caused failed logout will throw "Invalid session resource" error

Reply:
If we get to this point in the script, we're past the password change required exception case. So, we'd just need the try/except statement in the logout when catching that exception.

Although I do need to raise this observation with others; going over the Redfish spec, we don't call out allowing DELETE on your own session resource, which seems odd to me...